include/git2/version.h

```diff
@@ -7,12 +7,12 @@
 #ifndef INCLUDE_git_version_h__
 #define INCLUDE_git_version_h__
 
-#define LIBGIT2_VERSION "0.28.0"
+#define LIBGIT2_VERSION "0.99.0"
 #define LIBGIT2_VER_MAJOR 0
-#define LIBGIT2_VER_MINOR 28
+#define LIBGIT2_VER_MINOR 99
 #define LIBGIT2_VER_REVISION 0
 #define LIBGIT2_VER_PATCH 0
 
-#define LIBGIT2_SOVERSION 28
+#define LIBGIT2_SOVERSION "0.99"
```

Comment:
Hum. Any reason why we don't go all the way and say "0.99.0", matching LIBGIT2_VERSION?

Reply:
I don't think that we want our soversion to include teeny version numbers - IMO, we shouldn't be making ABI incompatible changes in teeny releases.  Are there other considerations that we should be thinking about?

With this change, we also get a nice set up of our library links on the filesystem:

```
 1852 -rwxr-xr-x   1 ethomson  staff  1895388 Feb  3 10:51 libgit2.0.99.0.dylib*
    0 lrwxr-xr-x   1 ethomson  staff       20 Feb  3 10:51 libgit2.0.99.dylib@ -> libgit2.0.99.0.dylib
    0 lrwxr-xr-x   1 ethomson  staff       18 Feb  3 10:51 libgit2.dylib@ -> libgit2.0.99.dylib
```